source/hwPortUtils.py

```diff
@@ -620,3 +488,60 @@ def listHidDevices(onlyAvailable: bool = True) -> typing.Iterator[dict]:
 
 	if _isDebug():
 		log.debug("Finished listing HID devices")
+
+
+_MOVED_SYMBOLS: dict[str, tuple[str, ...]] = {
+	"BLUETOOTH_ADDRESS": ("winBindings.bthprops",),
+	"BLUETOOTH_MAX_NAME_SIZE": ("winBindings.bthprops",),
+	"BTH_ADDR": ("winBindings.bthprops", "BLUETOOTH_ADDRESS"),
+	"CM_Get_Device_ID": ("winBindings.cfgmgr32",),
+	"CR_SUCCESS": ("winBindings.cfgmgr32",),
+	"MAX_DEVICE_ID_LEN": ("winBindings.cfgmgr32",),
+	"DEVPROPKEY": ("winBindings.setupapi",),
+	"dummy": ("winBindings.setupapi", "_Dummy"),
+	"PSP_DEVICE_INTERFACE_DATA": ("winBindings.setupapi",),
+	"PSP_DEVICE_INTERFACE_DETAIL_DATA": ("winBindings.setupapi",),
+	"PSP_DEVINFO_DATA": ("winBindings.setupapi",),
+	"SetupDiEnumDeviceInfo": ("winBindings.setupapi",),
+	"DIGCF_PRESENT": ("winBindings.setupapi", "DIGCF", "PRESENT"),
+	"DIGCF_DEVICEINTERFACE": ("winBindings.setupapi", "DIGCF", "DEVICEINTERFACE"),
+	"SPDRP_DEVICEDESC": ("winBindings.setupapi", "SPDRP", "DEVICEDESC"),
+	"SPDRP_HARDWAREID": ("winBindings.setupapi", "SPDRP", "HARDWAREID"),
+	"SPDRP_FRIENDLYNAME": ("winBindings.setupapi", "SPDRP", "FRIENDLYNAME"),
+	"SPDRP_LOCATION_INFORMATION": ("winBindings.setupapi", "SPDRP", "LOCATION_INFORMATION"),
+	"DICS_FLAG_GLOBAL": ("winBindings.setupapi", "DICS_FLAG", "GLOBAL"),
+	"DIREG_DEV": ("winBindings.setupapi", "DIREG", "DEV"),
+	"ERROR_NO_MORE_ITEMS": ("winAPI.constants", "SystemErrorCodes", "NO_MORE_ITEMS"),
+	"ERROR_INSUFFICIENT_BUFFER": ("winAPI.constants", "SystemErrorCodes", "INSUFFICIENT_BUFFER"),
+}
+"""Mapping from symbol name to new (absolute) module and symbol path."""
+
+
+def __getattr__(attrName: str) -> typing.Any:
```

Comment:
I like this new modular design. Do you think we could factor this out to a generic function we can just import and pass in the dict to? and update the deprecation docs to encourage this usage